runtime/astar/src/lib.rs

```diff
@@ -461,22 +470,25 @@ impl cumulus_pallet_parachain_system::Config for Runtime {
     type ReservedDmpWeight = ReservedDmpWeight;
     type XcmpMessageHandler = XcmpQueue;
     type ReservedXcmpWeight = ReservedXcmpWeight;
-    type CheckAssociatedRelayNumber = RelayNumberStrictlyIncreases;
+    type CheckAssociatedRelayNumber =
+        cumulus_pallet_parachain_system::RelayNumberMonotonicallyIncreases;
```

Comment:
Shouldn't we keep it as before until async backing is actually enabled?

It's not like we can build multiple parachain blocks in the context of a single relay chain block.

Reply:
I don't think ConsensusHook allows this but we can revert to strictly increase until we enable